bokeh/_testing/util/selenium.py

```diff
@@ -170,7 +170,8 @@ def enter_text_in_element(driver, element, text, click=1, enter=True, mod=None):
 def enter_text_in_cell(driver, cell, text):
     actions = ActionChains(driver)
     actions.move_to_element(cell)
-    actions.double_click()
+    actions.double_click() # start editing a cell
+    actions.double_click() # select all text and overwrite it in the next step
```

Comment:
I'm not sure if this is a change of behavior, a bug or something else, but getting certain editors to start editing is a bit finicky right now (reproducible when working manually with data tables).